README.md

```diff
@@ -39,21 +39,10 @@ Navigate to <http://localhost:8080>.
 
 ## Documentation
 
-### [Official docs](https://defectdojo.github.io/django-DefectDojo/)
-
-### [Getting Started](readme-docs/GETTING-STARTED.md)
-
-### [REST APIs](readme-docs/REST-APIs.md)
-
-### [Client APIs and Wrappers](readme-docs/CLIENT-APIs-AND_WRAPPERS.md)
-
-### [Release and Branch Model](readme-docs/RELEASE-AND-BRANCH-MODEL.md)
-
-### [Contributing](readme-docs/CONTRIBUTING.md)
-
-### [Roadmap](readme-docs/ROADMAP.md)
-
-### [Wishlist](readme-docs/WISHLIST.md)
+- [Official docs](https://defectdojo.github.io/django-DefectDojo/) ([latest](https://defectdojo.github.io/django-DefectDojo/) | [dev](https://defectdojo.github.io/django-DefectDojo/dev))
+- [Getting started](https://defectdojo.github.io/django-DefectDojo/usage/)
```

Comment:
```suggestion
- [Getting started](https://defectdojo.github.io/django-DefectDojo/getting_started/)
```

Reply:
This was copied from the original README I think the idea is to point users to the usage of defect dojo instead of the installation part which is behind the getting started link. I'm fine with your suggestion, but we might as well remove this line altogether then as the line above it already points to the docs with 'getting started' at the top of the sidebar menu?

Reply:
That's true, I think we schould remove the line.

Reply:
Yeah. If needed, we can improve the docs homepage.